plugins/hydra_nevergrad_sweeper/README.md

```diff
@@ -0,0 +1,57 @@
+# Hydra Nevergrad Sweeper plugin
```

Comment:
Plugins are documented in the website, with a link to the website from the readme.
See the other plugins readme for example.

To update the website, see website/README.md.
You basically start a local Docusaurus instance and then you can edit files under website/docs.

Let me know if have any questions.

Reply:
Don't forget to clean up the readme when you are done with the website. (see the joblib plugin for what a standard plugin readme can look like).